client/src/components/App.jsx

```diff
@@ -1,32 +1,112 @@
+/* eslint-disable import/extensions */
+/* eslint-disable no-undef */
 import React from 'react';
-// import ReactDOM from 'react-dom';
-
-const divStyle = {
-  color: 'white',
-  backgroundColor: 'grey',
-};
-
-const App = () => {
-  return (
-    <div style={divStyle}>HI IM WORKING!</div>
-  );
-};
-
-// class App extends React.Component {
-//   constructor(props) {
-//     super(props);
-
-//     this.state = {
-//     };
-//   }
-
-//   render() {
-//     return (
-//       <div style={divStyle}>
-//         <div>HI IM WORKING!</div>
-//       </div>
-//     );
-//   }
-// }
+import axios from 'axios';
+import styled from 'styled-components';
+import Price from './Price.jsx';
+import Dates from './Dates.jsx';
+import Guests from './Guests.jsx';
+import Fees from './Fees.jsx';
+import Button from './Button.jsx';
+
+const Boxed = styled.section`
+  padding: 2em;
+  background: white;
+  border: 1px solid;
+  border-color: lightgray;
+  border-radius: 1px;
+`;
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      price: {
+        price: 104,
+        rating: 4.96,
+        reviews: 191,
+      },
+      dates: {
+        startDate: 'Check-in',
+        endDate: 'Checkout',
+      },
+      guests: 4,
+      fees: {
+        price: 104,
+        days: 2,
+        cleaning: 100,
+        service: 43,
+        tax: 51,
+        total: 402,
+      },
+    };
+  }
+
+  componentDidMount() {
+    this.getData();
+  }
+
+  getData() {
+    const that = this;
+    const stay = that.state.fees.days;
+    console.log('i ran');
+    axios.get('/data')
+      .then(({ data }) => {
+        const res = data[0];
+        console.log(res);
+        const total = res.cleaning_fee + res.service_fee + res.tax + (res.price * stay);
+        this.setState({
+          price: {
+            price: res.price,
+            rating: res.rating,
+            reviews: res.reviews,
+          },
+          guests: res.guest_limit,
+          fees: {
+            price: res.price,
+            days: stay,
+            cleaning: res.cleaning_fee,
+            service: res.service_fee,
+            tax: res.tax,
+            total,
```

Comment:
@bregbromley, Everything LGTM. As for code cleanup, maybe to consider utilizing object destruction as stretch goal ?